packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -209,15 +210,10 @@ const NotificationDetail = () => {
       <TitleBox
         variantTitle="h4"
         title={notification.subject}
-        sx={{
-          pt: 3,
-          mb: {
-            xs: 3,
-            md: 4,
-          },
-        }}
+        sx={{ pt: 3, mb: 2 }}
```

Comment:
Fare attenzione al caso in cui l'abstract non sia presente. In questo caso gestire le spaziature con gli elementi circostanti